crates/vsock/README.md

```diff
@@ -43,6 +43,7 @@ Run the vhost-user-vsock device:
 vhost-user-vsock --guest-cid=<CID assigned to the guest> \
   --socket=<path to the Unix socket to be created to communicate with the VMM via the vhost-user protocol>
   --uds-path=<path to the Unix socket to communicate with the guest via the virtio-vsock device>
+  [--conn-tx-buf-size=<the size of the connection tx buffer>]
```

Comment:
What about `--tx-buffer-size=<size of the buffer used for the TX virtqueue (guest->host packets)>`?